src/Components/Components/src/ComponentBase.cs

```diff
@@ -48,16 +48,14 @@ public ComponentBase()
     protected ComponentPlatform Platform => _renderHandle.Platform;
 
     /// <summary>
-    /// Gets the target <see cref="IComponentRenderMode"/> this component will run in after prerendering.
+    /// Gets the <see cref="IComponentRenderMode"/> assigned to this component.
```

Comment:
This change is to avoid implying we are currently prerendering, or that prerendering is even applicable (since on WebView or standalone WebAssembly it wouldn't be applicable as a concept).